### PR TITLE
Finance page translation

### PR DESCRIPTION
**Pull request by dizsmek**
Go to both the normal finance page and the finance admin page and

- check the translation
- Toggle between languages and check if everything updates dynamically.

### CODE REVIEW

File: client/src/components/FinanceDashboard/FinanceDashboard.js [hunk 1]
```diff
@@ -1,46 +1,30 @@
...
+                return (
                     <div key = {key}>
-                        {`Current balance of ${key} pocket: ${value} ${currency}`}
+                        {BEFOREPOCKETNAME + financePockets[key.toUpperCase()] + AFTERPOCKETNAME + `${value} ${currency}`}
                     </div>
                 );
-            }
-            this.setState({ balances: categories });
-        } catch (error) {
-            this.props.onError(error);
```
Comment (DenesDanko) [inline client/src/components/FinanceDashboard/FinanceDashboard.js:61]: Let's ask Benji, what is the purpose of this try/catch & error handling here.
Maybe he knows something we can not understand right now.

**Comment by DenesDanko** [comment]
Heavy task Misu - especially with class->functional transformation on the fly! -, I am really glad you have the courage to take It!

'No user selected' translation is missing ATM, and the translation of the popups are also missing, right?
I guess they are in progress.

I am afraid merge will not be easy/trivial since FA page changed meanwhile! :o(

File: client/src/components/FinanceDashboard/FinanceDashboard.js (hunk 1 above)
Comment (dizsmek) [reply client/src/components/FinanceDashboard/FinanceDashboard.js:61]: @JoBenjamin en nem lattam hogy mi celbol van ott a try catch. Miert raktad bele?

**Comment by dizsmek** [comment]
> Heavy task Misu - especially with class->functional transformation on the fly! -, I am really glad you have the courage to take It!
> 
> 'No user selected' translation is missing ATM, and the translation of the popups are also missing, right?
> I guess they are in progress.
> 
> I am afraid merge will not be easy/trivial since FA page changed meanwhile! :o(

The 'No user selected' quote is translated at dictionary.userSelector. INVALIDUSERSELECTMESSAGE, but I changed the message a bit. I completely forgot about the popups... How about I do those in another PR?

I took a look at the merge conflict and it made me want to cry. But no need for tears, Benji refactored the UserSelector component almost the same way I did, so I'm just going to add the translation to his version again and throw away mine. Shouldn't take too long.

**Comment by dizsmek** [comment]
@DenesDanko could you do a final review on this so that I can merge? I'd like to merge this and do the translations of the popups on a different branch to avoid merge conflicts.

**Review by DenesDanko — APPROVED** [review]
Well, I am a bit uncertain to Approve, since you have removed two try/catch blocks.
And as Benji said, they were there for a good reason, although he could not remember why exactly, and I am not able to find out neither :o(

Otherwise the 'No user Selected' string is always in English.
Plus some very minor Lint errors..

How about this:
Fix Lint problems - that is easy - before merge.
We can add 'No user Selected' to the ticket #33 - Double check: all strings translation have to be done.

And If we met crashes we rethink the try/catch-es.

**Pull request merged**